stdlib/public/core/FixedPoint.swift.gyb

```diff
@@ -429,44 +426,71 @@ extension ${Self} {
 extension ${Self} : SignedNumber {}
 % end
 
-// Construction from other integer types
-@_transparent
-extension ${Self} {
-% for src_ty in all_integer_types(word_bits):
-%    srcBits = src_ty.bits
-%    srcSigned = src_ty.is_signed
-%    Src = src_ty.stdlib_name
-%    (srcSign, srcExt) = ('s', 'sext') if srcSigned else ('u', 'zext')
-%    if Self != Src:
-  public init(_ v: ${Src}) {
+
+%{
+import gyb
+
+fixed_fixed_conversion_function = gyb.parse_template("fixed_fixed_conversion_function",
+"""
 %
-    let srcNotWord = v._value
+  let srcNotWord = value._value
```

Comment:
If you feel like doing so, please feel free to rename `srcNotWord` to just `src`.  The `NotWord` part of the name is a leftover from an early implementation, it does not make any sense today.